webapp/src/Controller/SecurityController.php

```diff
@@ -103,58 +105,62 @@ public function registerAction(
         $registration_form->handleRequest($request);
         if ($registration_form->isSubmitted() && $registration_form->isValid()) {
             $plainPass = $registration_form->get('plainPassword')->getData();
-            $password  = $passwordHasher->hashPassword($user, $plainPass);
-            $user->setPassword($password);
-            if ($user->getName() === null) {
-                $user->setName($user->getUsername());
-            }
+            if (strlen($plainPass) < UserController::MIN_PASSWORD_LENGTH) {
+                $this->addFlash('danger', "Password should be " . UserController::MIN_PASSWORD_LENGTH . "+ chars.");
```

Comment:
```suggestion
                $this->addFlash('danger', "Password should be " . UserController::MIN_PASSWORD_LENGTH . "+ chars.");
                return $this->redirectToRoute('login');
```

Would make the PR so much easier to read I think.

All other changes are just indentations right?

Reply:
All remaining changes are indentation indeed, and I agree that here it makes a lot of sense to return early, not just to make the diff simpler, but more importantly to reduce code nesting complexity.

Reply:
We shouldnt return to login though, but register. will fix.